docs/docs/chitchat-faqs.mdx

```diff
@@ -2,336 +2,72 @@
 id: chitchat-faqs
 sidebar_label: Chitchat and FAQs
 title: Chitchat and FAQs
+abstract: FAQ assistants are the simplest assistants to build and typically the first kind of assistant anyone builds. This page is a guide to the concepts and training data you need to handle non-contextual questions like FAQs and chitchat.
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
+[FAQs](./glossary.mdx#faqs) and [chitchat](./glossary.mdx#chitchat) are two cases where the conversational assistant's answer is a single message,
+and the assistant should always answer the same way,
+no matter what has happened previously in the conversation.
+For example, in the
+following conversation, every question's answer is independent of context and
+could
+just as easily be answered on its own:
```

Comment:
is there a reason this is split over so many lines? Also, can we rephrase this to "For example, in the following conversation, every question can be asked at any point in the conversation and the answer is independent of anything the user has said previously" or similar